extensions/resource-deployment/src/services/tools/dockerTool.ts

```diff
@@ -46,4 +46,12 @@ export class DockerTool extends ToolBase {
 	protected get versionCommand(): string {
 		return 'docker version --format "{{json .}}"';
 	}
+
+	get autoInstallSupported(): boolean {
+		return false;
```

Comment:
When we list dependencies on tools we will need to add a columnt to that table to list if this is AutoInstallable or not, so that user knows what to expect when the user clicks on the install button.

Reply:
we don't have a scenario where you have mixed type of tools, you will either be able to install all or none at this point. existing message is good enough.